content/en/docs/Writing policies/tips.md

```diff
@@ -48,9 +48,12 @@ These are some tips and tricks you can use when putting together your Kyverno po
 * Ensure the resource you're matching and the spec definition align. For example, if writing a `mutate` rule which matches on a Deployment, the spec of what is being mutated needs to also align to a Deployment which may be different from, for example, a Pod. When copying-and-pasting from other rules, remember to check the spec.
 
 * Check Kyverno logs when designing rules if the desired result is not achieved:
-  ```sh
-  kubectl -n <kyverno_namespace> logs -l app=kyverno
-  ```
+
+```sh
+kubectl -n <kyverno_namespace> logs <pod_name>
```

Comment:
Could you explain the switch from a label selector to the pod name?

Reply:
Due to the inconsistencies that still exist between the YAML manifest method of installation and the Helm chart, the label `app=kyverno` may not exist (it's `app.kubernetes.io/name=kyverno` in Helm). Rather than putting another label, I just though I'd do away with that to make it simpler.

Reply:
Action item here at all?